### PR TITLE
Chapter 3 Challenge 1 (Solo-Lose)

### LINKED ISSUE #504: Chapter 3.1 - Solo vs Bitrey lesson (Lose)
Using the `HashrateChallenge` component we created in https://github.com/saving-satoshi/saving-satoshi/issues/503 implement the Co-op vs. Bitrey (Lose) challenge. In this challenge the player and cooperators will go up against Bitrey. They lose this round because they didn't split the nonces when working together.

[Link to Figma](https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?type=design&node-id=4102-52875&t=LZXGkrObhw4Swxjv-4)

### PR DESCRIPTION
**Pull request by benalleng**
Closes #504

Should finalize the solo-1 lesson

Adds text-fade and corrects some css classes to have consistent responsive design with the varying steps

[Preview](https://saving-satoshi-git-fork-benalleng-issue504-savingsatoshi.vercel.app/en/chapters/chapter-3/solo-1?dev=true)

#### WontFix
- click on hashrate to change format tip as shown in figma
  - I want to put this into a separate PR as this will require some substantial code refactoring if we want this feature everywhere

### CODE REVIEW

**Comment by benalleng** [comment]
> One thing though is that the design in figma has the text on the left for the solo challenge.

What do you mean by this? 

The start button should be glowing, it might be a browser specfic error, what browser are you using to test?

**Comment by ecurrencyhodler** [comment]
I'm using google chrome.

Here's a recording to show the button isn't glowing.

https://github.com/saving-satoshi/saving-satoshi/assets/32662508/55103be9-1a78-47f3-a916-ed4518c66a5c

**Comment by ecurrencyhodler** [comment]
Thanks for the fix Ben. I see that when the screen is wider, the text is on the left. Here's a video showing how it looks like as I dynamically make the window smaller.

https://github.com/saving-satoshi/saving-satoshi/assets/32662508/8bbbd3ed-736c-42f2-a477-c815b863c715

File: content/lessons/chapter-3/solo-1.tsx [hunk 1]
```diff
@@ -59,8 +77,43 @@ export default function Solo1({ lang }) {
...
+    if (step === 1) {
+      interval = setInterval(() => {
+        currentAntagonistHash =
+          currentAntagonistHash +
+          Math.floor(Math.random() * ANTAGONISTS[0].hashpower)
+        setAntagonistHash(currentAntagonistHash)
+      }, 40)
+    }
+    return () => clearInterval(interval)
+  }, [step])
```
Comment (afterburn) [inline content/lessons/chapter-3/solo-1.tsx:106]: You can combine this into a single `useEffect` since they have the same dependencies.

**Review by afterburn — CHANGES_REQUESTED** [review]
Requested small change

File: content/lessons/chapter-3/pool-1.tsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ export default function Pool1({ lang }) {
   const ANTAGONISTS = [
     {
       username: 'BitRey',
-      avatar: '/assets/avatars/5.png',
+      avatar: '/assets/avatars/bitrey.png',
```
Comment (afterburn) [inline content/lessons/chapter-3/pool-1.tsx:62]: I think for 'You' we should use the avatar of the currently logged in user (if it exists) and fall back to 1.png or perhaps a default profile icon or something:

image

**Comment by afterburn** [comment]
@benalleng I'm logged in here but it still shows 1.png while my avatar is set to 5.png:

Screenshot 2023-07-24 at 23 09 00

**Comment by benalleng** [comment]
> I'm logged in here but it still shows 1.png

@afterburn I wanted to confirm if you are still getting this behavior?  The code shouldn't fallback to 1.png ever, it should either be the standar icon if not logged in or your account avatar

**Review by afterburn — APPROVED** [review]

**Pull request merged**